### ISSUE TITLE
can this project select the specific version of BERT?

### ISSUE THREAD

**Issue by buptzcW** [huggingface/pytorch-transformers#1076]
## ❓ Questions & Help
I dont know if this project can select the version of BERT which I need. For example, i want use BERT-wwm not BERT-basic, what should i do？ Can you help me, plz.

**Comment by LysandreJik** [comment]
Hi. You can check the documentation about the different checkpoints available for each model [here](https://huggingface.co/pytorch-transformers/pretrained_models.html).

If you're looking for BERT whole word masking, there are the following pretrained models that might be of interest: `bert-large-uncased-whole-word-masking`, `bert-large-cased-whole-word-masking`, `bert-large-uncased-whole-word-masking-finetuned-squad` and `bert-large-cased-whole-word-masking-finetuned-squad`.

**Comment by buptzcW** [comment]
@LysandreJik Thanks for your advice. But, in my situation, I have to use my corpus to train a new BERT with whole word mask, so I cant use the pre-trained BERT model, what I should do in this situation?

**Comment by LysandreJik** [comment]
Training an entire BERT model from scratch takes a lot of resources, and we don't have any scripts/examples that show how to do it with our library.

You could look at [Microsoft's repository](https://github.com/microsoft/AzureML-BERT) that uses our implementation to pre-train/fine-tune BERT.

**Comment by buptzcW** [comment]
@LysandreJik notice that the bert had been updated by adding whole word mask, do you updating your pytorch-transformer with this trick when you convert BERT from tf to pytorch
